.github/workflows/push.yml

```
@@ -68,7 +66,29 @@ jobs:
           push: true
           tags: ghcr.io/${{ github.repository }}:${{ needs.branch_meta.outputs.sha }}
           labels: ${{ steps.docker_meta_img.outputs.labels }}
-
+      - name: Docker meta Service Name (file storage)
+        id: docker_meta_img_file_storage
+        uses: crazy-max/ghaction-docker-meta@v4
+        with:
+          images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=ref,event=branch,enable=false,priority=600
+            type=sha,enable=true,priority=600,prefix=
+          labels: |
+            org.opencontainers.image.title=schulcloud-file-storage
+      - name: Build and push ${{ github.repository }} (file storage)
```

you need an owe step to test if the image exist. Because only the image build for this file-storage- failed and not for the server image
```suggestion

      - name: test image exists (file storage)
        run: |
          mkdir -p ~/.docker 
          echo '{"experimental": "enabled"}' >> ~/.docker/config.json
          echo "IMAGE_EXISTS=$(docker manifest inspect ghcr.io/${{ github.repository }}:file-storage-${{ needs.branch_meta.outputs.sha }} > /dev/null && echo 1 || echo 0)" >> $GITHUB_ENV

      - name: Set up Docker Buildx (file storage)
        if: ${{ env.IMAGE_EXISTS == 0 }}
        uses: docker/setup-buildx-action@v2
          
      - name: Build and push ${{ github.repository }} (file storage)
```